source/documentation/concepts/security-controls.html.md.erb

```diff
@@ -12,10 +12,13 @@ systems.
 
 ## Network architecture
 
-The Cloud Platform (CP) is hosted in [Amazon AWS], and consists of two [kubernetes] clusters:
+The Cloud Platform (CP) is hosted in [Amazon AWS], and consists of three [kubernetes] clusters:
 
 * live-1 - the "application" cluster, managed via [Kops], which runs all hosted services
 * manager - an [EKS] cluster which runs shared services such as monitoring and CI/CD pipelines
+* live - The Cloud Platform team is asking all teams with services running on 
+the platform to migrate their services from live-1 to the new live cluster in the platform, during Autumn 2021.
+For information see [here](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/other-topics/migrate-to-live.html#migrate-to-the-quot-live-quot-cluster) 
```

Comment:
I think these two sentences seem a bit much - maybe compress them into this?
```* live - a newer "application" cluster, managed by EKS. Applications running on live-1 will [migrate to "live"](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/other-topics/migrate-to-live.html) during Autumn 2021.```

Reply:
Ah - an external link is failing for some reason now. Looking